packages_rs/nextclade/src/align/align.rs

```diff
@@ -66,16 +66,28 @@ pub fn align_nuc(
 
   let mut attempt = 0;
 
+  let mut pre_retry_alignment: Option<AlignmentOutput<Nuc>> = None;
+
   loop {
-    let stripes = create_alignment_band(
+    let stripes = match create_alignment_band(
       &seed_matches,
       qry_len as isize,
       ref_len as isize,
       terminal_bandwidth,
       excess_bandwidth,
       allowed_mismatches,
       params.max_band_area,
-    )?;
+    ) {
+      Ok(result) => result,
+      Err(err) => {
+        if let Some(alignment) = pre_retry_alignment {
+          info!("{err}");
+          info!("When processing sequence #{index} '{seq_name}': In nucleotide alignment on retry attempt {attempt}: Alignment area too large. Using previous alignment result. Alignment score was: {}", alignment.alignment_score);
+          return Ok(alignment);
+        }
+        return Err(err);
+      }
```

Comment:
This assumes that the "Alignment area too large" is the only kind of error that can be emitted by the `create_alignment_band()`. Which is currently true, but is not guaranteed. If another error is returned, the message will be inaccurate.

If we return `band_area` from `create_alignment_band()`, the check can be performed outside, so that we know what happened in this case. Also it will avoid the punctuational storm and nested scopes.